www/modules/centreon-open-tickets/lib/jquery.sheepItPlugin.js

```diff
@@ -160,20 +160,20 @@
                     ,nameTemplateAttr = getOrSetTemplate(that, "name")
                     ,idAttr = that.attr("id")
                     ,nameAttr = that.attr("name")
-                    
+
                 /* Normalize field name attributes */
-		if(typeof(newNameAttr)=='string'){
-                	newNameAttr = nameTemplateAttr.replace(options.indexFormat, index);
-                	that.attr("name", newNameAttr);
-		}
+        if(typeof(newNameAttr)=='string'){
```

Comment:
wrong JS indentation

Reply:
please add spaces after if, before first equal, after second equal and before opening the brace

Reply:
this is a code from an external lib (see the header), I tried not to change around in external libs. What will the policy be like? (including for license headers)

Reply:
this is a code from an external lib (see the header), I tried not to change around in external libs. What will the policy be like? (including for license headers)
CC: @julienmathis  @quanghungb